common/templates/layouts/base.njk

```diff
@@ -90,18 +90,17 @@
       }
     ]
   }) }}
+  {{ mojPrimaryNavigation({
+    label: 'Primary navigation',
+    items: [{
+      text: t("default::primary_navigation.dashboard_oca") if CURRENT_LOCATION and CURRENT_LOCATION.location_type === 'prison' else t("default::primary_navigation.dashboard_police"),
```

Comment:
What about rather than an if/else actually displaying both items based on permissions? No reason OCA shouldn't be able to see this in theory too. Makes this less complicated too.

Reply:
Both items link to `/moves` that then redirects accordingly. I am not entirely sure how to build the redirect logic if we want OCA to have access to the upcoming moves dashboard. Also, why would they want to? Upcoming moves would contain the active moves visible in the dashboard. That would generate confusion and not add anything.

Reply:
Hmm, yes maybe that's the tricky bit for now.

Ideally these nav items should deep link into the page that they are related too but I suppose our logic of the URLs containing a particular day (by default today or this week) make them trickier to build in a template file.